### PR TITLE
Revert "FFmpeg: Bump to 2.8.6-Jarvis-16.1"

### PR DESCRIPTION
**Pull request by fritsch**
This reverts commit 3e9e7d202b3a5b7dc16bbec7da1219cd6c9a6b4b.
Reason: It did not match the intended semantics - further discussion ongoing

### CODE REVIEW

File: tools/depends/target/ffmpeg/FFMPEG-VERSION [hunk 1]
```diff
@@ -1,5 +1,6 @@
 LIBNAME=ffmpeg
 BASE_URL=https://github.com/xbmc/FFmpeg/archive
-VERSION=2.8.6-Jarvis-16.1
+BRANCH=release/2.8-xbmc
```
Comment (FernetMenta) [inline tools/depends/target/ffmpeg/FFMPEG-VERSION]: having a "/" in the name does not work. needs to be something like 2.8-kodi

@wsnipex can we pull the sha from HEAD and trigger a re-build? currently depends just compares the names, right?

**Comment by fritsch** [comment]
jenkins build this please

File: tools/depends/target/ffmpeg/FFMPEG-VERSION (hunk 1 above)
Comment (fritsch) [inline tools/depends/target/ffmpeg/FFMPEG-VERSION]: I tried it on my local rig and it worked - afaik last time there was just a -p missing in the mkdir command.

Comment (MartijnKaijser) [inline tools/depends/target/ffmpeg/FFMPEG-VERSION]: Window will certainly don't like it

Comment (fritsch) [inline tools/depends/target/ffmpeg/FFMPEG-VERSION]: Oki - please cancel the PR - let's wait for wsnipex

**Comment by wsnipex** [comment]
https://github.com/wsnipex/xbmc/commit/5aa930d235a07a87d1de0a47cf5970cfc0d43bd3 fixes using those release/2.8-xbmc branch names in VERSION

**Comment by fritsch** [comment]
Thanks much I cherry-picked it and fixed the conflicts. Mmh Not sure it still works.

**Comment by fritsch** [comment]
Nope, that won't work as intended

**Comment by wsnipex** [comment]
its intended to be used with VERSION=release/2.8-xbmc not BRANCH.
afaik depends already worked with that, but didn't check

**Comment by wsnipex** [comment]
tested depends and it works.
to clarify: revert the Makefile change, remove BRANCH and set VERSION=release/2.8-xbmc

**Comment by fritsch** [comment]
@wsnipex did I understand correctly? ^^

**Comment by wsnipex** [comment]
yep, fine. Lets see if it works on windows as well...
jenkins build this please

**Comment by FernetMenta** [comment]
does this work with label and branch?

**Comment by wsnipex** [comment]
it should work with all valid git refs.

**Comment by afedchin** [comment]
windows doesn't like '/' in filenames.

**Comment by wsnipex** [comment]
linux doesn't either ;)
here is an attempt at fixing win32 as well: https://github.com/wsnipex/xbmc/commit/105958c88212cef81853b68a1b6af91b90d1db1c

will run a jenkins test on it.

**Comment by wsnipex** [comment]
strange, anyone knows whats going on on win32: http://jenkins.kodi.tv/view/Windows/job/WIN-32/7766/console
this is with the patch to buildffmpeg.sh from above. @Paxxi maybe?

**Comment by afedchin** [comment]
@wsnipex seems `$UNZIP e -y $ARCHIVE` extracts all files without folders at all. 

```
 e: Extract files from archive (without using directory names)
```

**Comment by wsnipex** [comment]
jenkins build this please

**Pull request merged**